packages/@divvi/mobile/src/walletConnect/saga.ts

```diff
@@ -504,6 +504,37 @@ function* showActionRequest(request: WalletKitTypes.EventArguments['session_requ
     return
   }
 
+  if (method === SupportedActions.wallet_getCapabilities) {
+    const [address, requestedChainIds] = request.params.request.params
+    if (!address) {
+      yield* put(denyRequest(request, { code: -32602, message: 'Invalid params' }))
```

Comment:
nit: do these error codes have semantic meaning? can we name them/put them in an enum in `walletConnect/constants.ts`? not a big deal